### PR TITLE
fixed signup form

### LINKED ISSUE #358: signup form squeezed
image

all the items of signup form looked squeezed with no margin, padding on the page https://accounts.susi.ai/signup.

### PR DESCRIPTION
**Pull request by pg07codes**
fixes #358

sign-up form now looks like -

image

### CODE REVIEW

**Comment by pg07codes** [comment]
@PrP-11 -fixed signup page
- surge deployl link : lyrical-canvas.surge.sh
-did not fix #357 in this . The staticAppBar component is used on both the home and the /signup page but behaves differently on scrolling , i was unable to find any reasons

image

**Comment by PrP-11** [comment]
This is what it appears for me. 

login

**Comment by pg07codes** [comment]
Now it is fixed. the other thing that you point out about the footer is intentional . i've added margin to the image so signup form does not look too stuffed.
- would request you to see the changes in PR deploy link: ```violent-spoon.surge.sh```

**Comment by pg07codes** [comment]
@PrP-11 
- would appreciate if you can resolve my problem. Would love to login and work on the issues there. #371

**Comment by love-san** [comment]
@pg07codes I see that the issue has been open for quiet a while now. 
If you don't mind can I fix the signup page. Thanks.

**Comment by pg07codes** [comment]
@love-san its been nearly fixed now. was quiet busy so unable to make a pull request.

**Comment by pg07codes** [comment]
@PrP-11 please review sir.

**Comment by pg07codes** [comment]
@PrP-11 fixed the background color. please merge this. Will create a separate pull for #357(unable to find the problem as of now)
deploy link : http://aboard-chicken.surge.sh

**Comment by pg07codes** [comment]
i think i messed up something while rebasing(squashing). Will clear the PR by tommorow

**Comment by PrP-11** [comment]
@pg07codes Please follow up on this PR. This is urgently needed.

**Comment by pg07codes** [comment]
@PrP-11 Sure. By today, i will.

**Comment by AnupKumarPanwar** [comment]
@pg07codes Resolve the conflicts.

**Comment by pg07codes** [comment]
resolved the conflicts. sorry, i took too long. @PrP-11 , @anshumanv , @AnupKumarPanwar please verify

**Comment by pg07codes** [comment]
done @fragm3 . please merge it now @anshumanv

File: src/components/Auth/ForgotPassword/ForgotPassword.react.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ import ChatConstants from '../../../constants/ChatConstants';
 
 import { urls } from '../../../Utils';
 
+import urls from '../../../utils/urls';
```
Comment (DNS-404) [inline src/components/Auth/ForgotPassword/ForgotPassword.react.js]: You are importing `urls` again here.

File: src/components/Auth/Login/Login.react.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ import susi from '../../../images/susi-logo.svg';
 import { urls, isProduction } from '../../../Utils';
 import ChatConstants from '../../../constants/ChatConstants';
 
+import urls from '../../../utils/urls';
```
Comment (DNS-404) [inline src/components/Auth/Login/Login.react.js]: Same.

**Comment by pg07codes** [comment]
@DNS-404 ok. i am closing this PR and will make another PR after fetching all the changes.

**Pull request closed without merging**